mixer/adapter/list/regexList.go

```diff
@@ -55,10 +55,8 @@ func parseRegexList(buf []byte, overrides []string) (*regexList, error) {
 	}
 
 	for _, override := range overrides {
-		exp, err := regexp.Compile(override)
-		if err != nil {
-			return nil, err
-		}
+		// cannot fail, override syntax was checked in the Validate method
+		exp, _ := regexp.Compile(override)
```

Comment:
is this really better code ? sometimes 100% coverage is worse than 99%
(like something isn't supposed to happen, until someone changes Validate and miss something and now it fails silently in production)

Reply:
There are tests to ensure Validate is doing the right thing.